packages/account/src/BaseSmartAccount.ts

```diff
@@ -242,8 +242,8 @@ export abstract class BaseSmartAccount implements IBaseSmartAccount {
       delete userOp.maxFeePerGas;
       delete userOp.maxPriorityFeePerGas;
       // Making call to bundler to get gas estimations for userOp
-      const { callGasLimit, verificationGasLimit, preVerificationGas, maxFeePerGas, maxPriorityFeePerGas } =
-        await this.bundler.estimateUserOpGas(userOp);
+      const { callGasLimit, verificationGasLimit, preVerificationGas, maxFeePerGas, maxPriorityFeePerGas, paymasterAndData } =
+        await this.paymaster.estimateUserOpGas(userOp);
```

Comment:
won't this be done as part of pm_sponsorUserOperation later anyways?
I thought the point was removing any estimations (default behavior) in this function call